app/components/MultiDataProgressBar/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { NumberOutput } from '@the-deep/deep-ui';
+
+import styles from './styles.css';
+
+export interface ProgressBarProps {
+    className?: string | undefined;
+    barHeight: number;
+    suffix?: string;
+    progressInfoData: {
+        source: string;
+        id: string;
+        country: number;
+        regional: number;
+        totalValue: number;
+    };
+}
+
+function MultiDataProgressBar(props: ProgressBarProps) {
+    const {
+        className,
+        barHeight,
+        progressInfoData,
+        suffix,
+    } = props;
+
+    const countryPercentage = useMemo(
+        () => (
+            {
+                percentage: (
+                    (progressInfoData.country / progressInfoData.totalValue) * 100).toFixed(0),
+            }
+        ), [progressInfoData],
+    );
+
+    const regionalPercentage = useMemo(
+        () => (
+            {
+                percentage: (
+                    (progressInfoData.regional / progressInfoData.totalValue) * 100).toFixed(0),
+            }
+        ), [progressInfoData],
+    );
+
+    const tooltip = () => {
+        if (progressInfoData?.country
+            && progressInfoData?.regional
+            && progressInfoData?.totalValue) {
+            return (
+                `Country: ${progressInfoData.country ?? '0'}, Regional: ${progressInfoData.regional ?? '0'}`
+            );
+        }
+        return '';
+    };
+
+    return (
+        <div className={_cs(className, styles.progressInfo)}>
+            <div className={styles.progressTitle}>
+                {progressInfoData.source}
```

Comment:
I think what you're referring to as 'source' is an indicator title. Let's use appropriate naming.

Reply:
done